CMakeLists.txt

```diff
@@ -371,7 +370,7 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/cmake-zipconf.h.in ${CMAKE_CURRENT_BI
 
 SET(srcdir ${CMAKE_CURRENT_SOURCE_DIR}/regress)
 SET(abs_srcdir ${CMAKE_CURRENT_SOURCE_DIR}/regress)
-SET(top_builddir ${CMAKE_BINAfindRY_DIR})
+SET(top_builddir ${CMAKE_BINARY_DIR})
```

Comment:
In retrospect, I probably shouldn't have tried to build this patch using github's text editor...